content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -36,3 +42,7 @@ Corda supports the following `SignatureSpecs` (signature schemes) for creating `
 * GOST3411withGOST3410
 
 Use `SignatureSpecService` to retrieve the `SignatureSpec`.
+
+Initially the above list of signature spec was available to users. However, they could be passing the wrong signature spec for a signing key type (e.g. an RSA key with an SHA256withECDSA), which would lead to an error when attempting to generate the signature.
```

Comment:
```suggestion
Initially the above list of signature spec was available to users. However, they could be passing the wrong signature spec for a signing key type (for example, an RSA key with an SHA256withECDSA), which would lead to an error when attempting to generate the signature.
```